cmd/security-secretstore-setup/Dockerfile

```diff
@@ -55,4 +55,8 @@ COPY --from=builder /edgex-go/cmd/security-secretstore-setup/entrypoint.sh /usr/
 RUN chmod +x /usr/local/bin/entrypoint.sh \
     && ln -s /usr/local/bin/entrypoint.sh /
 
+# Create token directory and assign appropriate permissions
+RUN mkdir -p /vault/config/assets && \
+    chown -Rh 100:1000 /vault/
+
```

Comment:
minor suggestion: reduce number of docker image layers by combining this with the above run.

Reply:
That actually does run in a single "RUN" command (and thus a single image layer) - the line is just split for readability.

Reply:
no i meant with the above RUN, not just this one

Reply:
Check it again - I think I get what you are saying. I combined the two RUN commands so it creates 1 image layer for those 4 commands instead of 2 image layers.

Reply:
cool